src/entity.rs

```diff
@@ -96,6 +102,24 @@ impl From<i64> for DatastoreValue {
     }
 }
 
+impl From<Vec<String>> for DatastoreValue {
+    fn from(str_vec_value: Vec<String>) -> DatastoreValue {
+        fn generate_string_value(str_val: String) -> Value {
+            let mut val = DatastoreValue::default();
+            val.string(str_val);
+            val.0
+        }
+        let mut array_val = DatastoreValue::default();
+        array_val.array(
+            str_vec_value
+                .into_iter()
+                .map(|s| generate_string_value(s))
```

Comment:
no need to create an "expensive" closure, just use fn? `.map(generate_string_value`?

Reply:
probably going to get optimized away, but anyway...

Reply:
ah! did not know, but when you say it sounds obvious, that I will start doing when possible